pkg/diagnose/connectivity/hooks.go

```diff
@@ -53,45 +53,40 @@ func createDiagnoseTrace(writer io.Writer) *httptrace.ClientTrace {
 	}
 }
 
-var (
-	// color functions are defined here to keep consistency when displaying information as it might be
-	// easier to read all the information about a specific section when the information concerning this
-	// section are colored with the same color.
-	dnsColorFunc  = color.MagentaString
-	tlsColorFunc  = color.YellowString
-	hintColorFunc = color.CyanString
-)
-
-// writeWrapper is used to pass a io.Writer variable to the hooks so that they
+// writeWrapper is used to pass slice a io.Writer variable to the hooks so that they
```

Comment:
Is this comment still valid ? Seems to talk about something else.

Reply:
Addressed with the changes from above comment